UnityProject/Assets/Scripts/ChemistryComponents/ReagentContainer.cs

```diff
@@ -502,7 +502,7 @@ private void NotifyPlayersOfSpill(Vector3Int worldPos)
 			}
 			else
 			{
-				Chat.AddLocalMsgToChat($"The {gameObject.ExpensiveName()}'s contents spill all over the floor!", gameObject);
+				Chat.AddActionMsgToChat(gameObject, $"The {gameObject.ExpensiveName()}'s contents spill all over the floor!");
```

Comment:
does this also display on top of players head?

Reply:
No